hummingbot/connector/exchange/gate_io/gate_io_api_order_book_data_source.py

```diff
@@ -56,29 +61,43 @@ def _get_throttler_instance(cls) -> AsyncThrottler:
         return throttler
 
     @classmethod
-    async def get_last_traded_prices(
-        cls, trading_pairs: List[str], throttler: Optional[AsyncThrottler] = None
-    ) -> Dict[str, Decimal]:
-        throttler = throttler or cls._get_throttler_instance()
+    async def get_last_traded_prices(cls, trading_pairs: List[str]) -> Dict[str, Decimal]:
+        throttler = cls._get_throttler_instance()
+        rest_delegate = await build_gate_io_api_factory().get_rest_delegate()
```

Comment:
That's the case for `get_order_book_data`, but only because that one is called from within the `GateIoAPIOrderBookDataSource` class and it then passes the `self.` of those parameters. However, this method is only called form `hummingbot/core/utils/market_price.py:27` and `hummingbot/core/data_type/order_book_tracker.py:161`. And in both cases, the generic signature of the method is used and those parameters are not passed, so I omitted them to be more explicit